src/style/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  
+  &__logo {
+    height: 40px;
+    width: 40px;
+    margin: 10px 0;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+
+  .is-active {
+    color: #00acdc;
+
+    &::after {
+      position: absolute;
+      content: "";
+      width: 37px;
```

Comment:
better not to set precise value, but to just make it spread over available space